draft-irtf-cfrg-hpke.md

```diff
@@ -347,21 +347,28 @@ A set of algorithm identifiers for concrete instantiations of these
 primitives is provided in {{ciphersuites}}.  Algorithm identifier
 values are two bytes long.
 
-The following two functions are defined for a KDF to facilitate domain
-separation of calls as well as context binding:
+The following two functions are defined to facilitate domain separation of
+KDF calls as well as context binding:
 
 ~~~
 def LabeledExtract(salt, label, IKM):
-  labeledIKM = concat("RFCXXXX ", label, IKM)
+  labeledIKM = concat("RFCXXXX ", suite_id, label, IKM)
   return Extract(salt, labeledIKM)
 
 def LabeledExpand(PRK, label, info, L):
-  labeledInfo = concat(I2OSP(L, 2), "RFCXXXX ", label, info)
+  labeledInfo = concat(I2OSP(L, 2), "RFCXXXX ", suite_id, label, info)
   return Expand(PRK, labeledInfo, L)
 ~~~
 
 \[\[RFC editor: please change "RFCXXXX" to the correct number before publication.]]
 
+The value of `suite_id` depends on where the KDF is used; it is assumed
+implicit from the implementation and not passed as parameter. If used
+inside a KEM algorithm, `suite_id` MUST start with "KEM" and identify
+this KEM algorithm; if used in the remainder of HPKE, it MUST start with
+"DEM" and identify the entire ciphersuite in use. See sections {{dhkem}}
```

Comment:
Nit: I would use "HPKE" for the non-KEM thing.

Reply:
Good point – I just made this change.